test/all.spec.ts

```diff
@@ -109,4 +109,96 @@ describe('validation', () => {
 		expect(f).to.throw('Invalid label name: "mal_formed"');
 		done();
 	});
+
+	it('should fail if a relative bind mount is specified', () => {
+		const f = () => {
+			compose.normalize({
+				version: '2.1',
+				services: {
+					main: {
+						image: 'some/image',
+						volumes: [
+							'./localPath:/some-place',
+						],
+					},
+				},
+			});
+		};
+		expect(f).to.throw('Bind mounts are not allowed');
```

Comment:
Shouldn't you be calling `done()` in the end?

Reply:
@dfunckt you only need to call it if you accept it as a paramater: https://mochajs.org/#synchronous-code